src/games/strategy/triplea/util/TransportUtils.java

```diff
@@ -198,4 +213,18 @@ public int compare(final Unit o1, final Unit o2) {
     return Optional.empty();
   }
 
+  private static void loadMaxUnits(final Unit transport, final List<Unit> canBeTransported,
```

Comment:
Agree that its not great but I don't think there is a good way to really make it an input/output method while still trying to pull the low level details out of the calling method. I think keeping the calling method simple and clean is more important in this case.